Source/WebKit/GPUProcess/media/RemoteMediaResourceManager.h

```diff
@@ -53,14 +53,14 @@ namespace WebKit {
 class RemoteMediaResource;
 
 class RemoteMediaResourceManager
-    : public IPC::WorkQueueMessageReceiver {
+    : public IPC::WorkQueueMessageReceiver<WTF::DestructionThread::Any> {
     WTF_MAKE_TZONE_ALLOCATED(RemoteMediaResourceManager);
 public:
     static Ref<RemoteMediaResourceManager> create() { return adoptRef(*new RemoteMediaResourceManager()); }
     ~RemoteMediaResourceManager();
 
-    void ref() const final { IPC::WorkQueueMessageReceiver::ref(); }
-    void deref() const final { IPC::WorkQueueMessageReceiver::deref(); }
+    void ref() const final { IPC::WorkQueueMessageReceiver<WTF::DestructionThread::Any>::ref(); }
```

Comment:
I'm pretty sure the template parameters aren't necessary.  I see a lot of RefCounted::ref existing and don't see why this would be different.  Same throughout the PR.

Reply:
I wish that were true.